config.js

```diff
@@ -1776,6 +1776,9 @@ var config = {
     //         collectionInterval?: number;
     //         logGetStats?: boolean;
     // },
+
+    // Hide login button on auth dialog, you may want to enable this if you are using JWT tokens to authenticate users
+    hideLoginButton: true,
```

Comment:
Please comment this out, it's not a good default.

Reply:
Of course sorry about that.